### PR TITLE
Add traefik dependancy

### PR DESCRIPTION
**Pull request by maulikjs**
## Related Issues and Dependencies
https://issues.redhat.com/browse/ODH-421

…

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

This Pull request installs the jupyterhub-traefik-proxy which will be used to write the routes to a .toml file in a configmap and connect to the external Traefik proxy.

### CODE REVIEW

**Comment by LaVLaS** [comment]
/lgtm

I can't confirm that it works but the changes appear to be valid

File: .jupyter/jupyterhub_config.py [hunk 1]
```diff
@@ -235,4 +233,20 @@ def clean_environment(spawner):
...
+c.TraefikTomlConfigmapProxy.traefik_api_url = "http://traefik-proxy:8099"
+
+# traefik api endpoint login username
+c.TraefikTomlConfigmapProxy.traefik_api_username = os.environ['TRAEFIK_API_USERNAME']
+# traefik api endpoint login password
+c.TraefikTomlConfigmapProxy.traefik_api_password = os.environ['TRAEFIK_API_PASSWORD']
+
+c.TraefikTomlConfigmapProxy.cm_namespace = os.environ['NAMESPACE']
+c.TraefikTomlConfigmapProxy.cm_name = "traefik-rules"
+c.TraefikTomlConfigmapProxy.traefik_svc_namespace = os.environ['NAMESPACE']
```
Comment (vpavlin) [inline .jupyter/jupyterhub_config.py:252]: Missing new line

**Comment by vpavlin** [comment]
lgtm

**Pull request merged**